assets/templates/search_results_layout_template.html

```diff
@@ -89,10 +89,7 @@ <h2 class="accordion-header border">
           >
             <div class="accordion-body">
               <div class="btn-group" role="group">
-                <button
-                  id="like-{{$i}}"
-                  class="btn btn-sm btn-outline-primary"
-                >
+                <button id="like-{{$i}}" class="btn btn-sm btn-outline-primary">
                   <i class="fa fa-thumbs-o-up"></i>
```

Comment:
do we get any other free options at font-awesome, for thumbs up ?

Reply:
There are animations that can be added such as `fade` or `Beat Fade` that you might want to try .

Reply:
> do we get any other free options at font-awesome, for thumbs up ?

We are currently using font-awesome v4.4, the latest has more options: https://fontawesome.com/icons/thumbs-up?f=classic&s=solid